### PR TITLE
Readme update 

### PR DESCRIPTION
**Pull request by lauraporta**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,20 +44,20 @@ For each one, type your answer, enter a single number (or just hit return) to ch
...
-	├── pyproject.toml
-	├── tests
-    │   ├── __init__.py
-    │   ├── test_integration
-    │   │   └── __init__.py
-    │   └── test_unit
-    │       ├── __init__.py
-    │       └── test_placeholder.py
-	└── tox.ini
+	LICENSE
```
Comment (adamltyson) [inline README.md:47]: It seems a lot harder to me to parse this directory structure without the lines.

File: README.md [hunk 2]
```diff
@@ -180,24 +180,7 @@ In the case you see `mypy` failing with an error like `Library stubs not install
...
-* `bump2version major`, e.g. bump from `0.0.1` to a release candidate `1.0.0rc0`
-* `bump2version rc`, e.g. bump from `0.1.0rc0` to create a new release candidate `0.1.0rc1`
-* `bump2version release`, e.g. bump from `0.1.0rc0` to create a new release `0.1.0`
-
-To ensure the tags are pushed to GitHub:
-
-```bash
-git push --follow-tags
-```
+Use setuptools_scm to automatically version your package. It has been pre-configured in the `pyproject.toml` file.
```
Comment (adamltyson) [inline README.md:183]: Can we add some basic info about how to use `setuptools_scm` here?

**Review by adamltyson — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (lauraporta) [reply README.md:183]: I added some info on versioning, however it is not yet clear to me how to move to a new version with this tool. 
In the [README](https://github.com/pypa/setuptools_scm/blob/main/README.rst), they put as examples python snippets, however I am not sure if they're meant to be used with pyproject.toml

**Comment by adamltyson** [comment]
I've updated the use of `setuptools_scm` to match the course materials, and I'm happy to merge if you are @lauraporta?

**Comment by lauraporta** [comment]
Yes, that's ok for me!

**Review by adamltyson — APPROVED** [review]

**Pull request merged**